cli/sync_service/README.md

```diff
@@ -0,0 +1,362 @@
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "link": "",
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "inactive": false,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "activationTime": "",
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "noData": false,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "metaOnly": false,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "destinationDataUri": "",
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "sourceDataUri": "",
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "consumers": 1,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "autodelete": false,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "originID": "Cloud",
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "originType": "Cloud",
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "deleted": false,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "instanceID": 1578587080905,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "dataID": 1578587080905,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "objectSize": 6,
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:     "chunkSize": 122880
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]:   }
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]: ]
+    Jan  9 16:25:05 lily-test workload-dcaaee7bd128e1820c79835c6e097277d768986cac6762a41ec0861663fa1fb2_amd64_usehello[25429]: Got a new object: lily1
+
+#### List MMS object
```

Comment:
Once you have the example described above and all the concepts, the following is the right kind of info to present.